cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -101,83 +101,83 @@ This section comprises a glossary of all the keywords -- grouped by category and
+|<<functions-haversin, haversin()>>             | Trigonometric     | Returns half the versine of a number.
 |<<functions-head,head()>>                      | Scalar            | Returns the first element in a list.
-|<<functions-id,id()>>                          | Scalar            | Returns the id of the relationship or node.
-|<<functions-keys,keys()>>                      | List              | Returns a list of string representations for the property names of a node, relationship, or map.
-|<<functions-labels,labels()>>                  | List              | Returns a list of string representations for the labels attached to a node.
+|<<functions-id,id()>>                          | Scalar            | Returns the id of a relationship or node.
+|<<functions-keys,keys()>>                      | List              | Returns a list containing the string representations for all the property names of a node, relationship, or map.
+|<<functions-labels,labels()>>                  | List              | Returns a list containing the string representations for all the labels of a node.
 |<<functions-last,last()>>                      | Scalar            | Returns the last element in a list.
-|<<functions-left,left()>>                      | String            | Returns a string containing the left n characters of the original string.
+|<<functions-left,left()>>                      | String            | Returns a string containing the specified number of leftmost characters of the original string.
 |<<functions-length,length()>>                  | Scalar            | Returns the length of a path.
-|<<functions-log, log()>>                       | Logarithmic       | Returns the natural logarithm of the expression.
-|<<functions-log10, log10()>>                   | Logarithmic       | Returns the common logarithm (base 10) of the expression.
-|<<functions-ltrim,lTrim()>>                    | String            | Returns the original string with whitespace removed from the left side.
-|<<functions-max,max()>>                        | Aggregating       | Returns the highest value in a numeric column.
-|<<functions-min,min()>>                        | Aggregating       | Returns the lowest value in a numeric column.
-|<<functions-nodes,nodes()>>                    | List              | Returns all nodes in a path.
-|<<functions-none,none()>>                      | Predicate         | Returns true if the predicate holds for no element in the list.
+|<<functions-log, log()>>                       | Logarithmic       | Returns the natural logarithm of a number.
+|<<functions-log10, log10()>>                   | Logarithmic       | Returns the common logarithm (base 10) of a number.
+|<<functions-ltrim,lTrim()>>                    | String            | Returns the original string with leading whitespace removed.
+|<<functions-max,max()>>                        | Aggregating       | Returns the maximum value in a set of values.
```

Comment:
I thought max was an aggregating function, but the new description seems to imply it takes a single list argument and returns the max element, which is not what it does. It should take a column identifier (or variable) and aggregated the column (reducing cardinality) to return a single value (max value), based on any other grouping conditions.